### PR TITLE
Simplify dashboard tag templates

### PR DESCRIPTION
**Pull request by makoncline**
## Summary

Replace the overdesigned dashboard tag layout builder with a preset-first workflow that stays simple for common tags and still supports custom layouts through a lightweight text syntax.

Users can now start from Simple name, Garden ID, Sale tag, or Grower details; customize rows with field tokens and small formatting markers; copy complete AI instructions; and keep QR codes fixed on the right while text stays on one line and shrinks to fit. Blank lines create intentional row spacing, common hybridizer/year output is handled cleanly, and saved custom templates have predictable save, discard, and delete behavior.

## Why

The previous cell-by-cell layout editor exposed too much implementation detail and made a common printing task feel like a page designer. This change makes the default path template selection, while retaining a narrow escape hatch for users who need custom content.

## File-by-file

- `AGENTS.md`: records the reusable rule for running concurrent Atlas flows on separate ports.
- `apps/main/playwright.atlas.config.ts`: registers the tag-printing Atlas test in the authenticated member project.
- `apps/main/scripts/atlas-flows.mjs`: declares the tag-printing flow, its visual states, and reproduction commands.
- `apps/main/scripts/run-atlas-flow.mjs`: lets Atlas use an explicitly selected free port while retaining its disposable database/server lifecycle.
- `apps/main/src/app/dashboard/tags/_components/tag-cell-editor.tsx`: removes the obsolete cell-by-cell editor.
- `apps/main/src/app/dashboard/tags/_components/tag-designer-html.ts`: aligns downloadable/print HTML with the no-wrap, shrink-to-fit, weighted-column, spacer-row, and fixed-right QR behavior.
- `apps/main/src/app/dashboard/tags/_components/tag-designer-model.ts`: defines the built-in templates, lightweight custom syntax, field catalog, AI instructions, legacy-state migration, spacer behavior, and resolved preview rows.
- `apps/main/src/app/dashboard/tags/_components/tag-designer-panel.tsx`: replaces the complex editor UI with template cards, one custom text editor, saved-template controls, a single AI-instructions entry point, and shared preview cards.
- `apps/main/src/app/dashboard/tags/_components/use-tag-designer-controller.ts`: simplifies template selection/customization state and makes save, discard, and delete behavior deterministic.
- `apps/main/tests/atlas/tag-printing.atlas.ts`: captures populated Simple name, Garden ID, Sale tag, Grower details, custom-template, AI-instructions, and sheet-creator states.
- `apps/main/tests/tag-designer-model.test.ts`: covers presets, parsing, combined fields, no-wrap migration, optional empty values, blank-line spacing, and column sizing.
- `apps/main/tests/tag-designer-panel.test.tsx`: covers the preset-first UI, customization, AI instructions, saved templates, and delete/discard edge cases.

## User impact

Most growers can choose a good template and print immediately. Advanced users can still express custom rows and columns without learning a visual layout system, and all selected listings render consistently across preview, print, PDF, and image output.

## Validation

- `pnpm exec vitest run tests/tag-designer-downloads.test.ts tests/tag-designer-model.test.ts tests/tag-designer-panel.test.tsx tests/tag-print-table.test.ts tests/atlas-flows.test.ts` — 54 tests passed
- `pnpm typecheck`
- focused ESLint on changed tag-designer and Atlas files
- focused Prettier check
- `git diff origin/main...HEAD --check`
- `ATLAS_PORT=3216 node scripts/run-atlas-flow.mjs tag-printing --output=../../local/atlas/tag-printing-pr` — 2 Playwright tests passed across 7 visual states

### CODE REVIEW

File: apps/main/src/app/dashboard/tags/_components/tag-designer-model.ts [hunk 1]
```diff
@@ -409,147 +348,77 @@ function createTemplateCell(
...
+    id: `${idPrefix}-${index}`,
+  }));
+}
+
+export const DEFAULT_TAG_DESIGNER_STATE: TagDesignerState = {
+  sizePresetId: "brother-tze-1",
+  customWidthInches: 3.5,
+  customHeightInches: 1,
+  showQrCode: true,
+  rows: createBuiltinTemplateRows("garden-id-row", GARDEN_ID_TEMPLATE),
```
[the remark this answers is not in the document]
Comment (makoncline) [reply apps/main/src/app/dashboard/tags/_components/tag-designer-model.ts:387]: Fixed in `0a69604f`. `sanitizeTagDesignerState` now detects the exact previous default row signature and migrates only those rows to the new Garden ID layout while preserving the stored size preset, custom dimensions, and QR preference. Added a regression test for that legacy state. Verification: 55 focused tests passed, plus typecheck, focused ESLint, Prettier, and `git diff --check`.
